wayland-sys/build.rs

```diff
@@ -0,0 +1,32 @@
+use pkg_config::Config;
+
+fn main() {
+    let features = std::env::vars()
+        .filter_map(|(key, _value)| {
+            let feature = key.trim_start_matches("CARGO_FEATURE_");
+            if key.contains("CARGO_FEATURE_") {
+                Some(feature.to_lowercase().replace('_', "-"))
+            } else {
+                None
+            }
+        })
+        .collect::<Vec<_>>();
```

Comment:
Given each feature is checked only once, what's the gain of creating this `Vec` instead of just checking `std::env::var_os("CARGO_FEATURE_FOO").is_some()` at each `if` lower?

Reply:
kinda more convenient to check using lowercase names, but sure, let's do the simpler thing